### PR TITLE
Feature/refactor part1

### PR DESCRIPTION
**Pull request by KarrarQ**
## What does this merge do? Check all that apply.
- [X] Adds a new feature
- [X] Improves an existing feature
- [X] Fixes a bug
- [X] Cleans up (logs, tests, etc.)

## Description:
Briefly, but clearly describe what you did, and explain any new code.

## Issues:
Are there any concerns, issues, or bugs in this branch? If so describe them.
 
Fixed restrictCalendarRangeMin && Max can set the min and max within one function
Fixed indentions - lines: 201 - 203
Added sad path to calcuateSleepByDate() testing 
remove unused variables from hydro repo test 

## Checklist

- [X] My code follows the style guidelines of this project
- [X] I have performed a self-review of my own code
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] I have used tests to prove my fix is effective or that my feature works
- [X] New and existing unit tests pass locally with my change

### CODE REVIEW

**Review by BHaddad1 — APPROVED** [review]
Looks really good, thanks!

File: src/scripts.js [hunk 1]
```diff
@@ -139,15 +138,10 @@ function restrictCalendarRangeMin() {
   const min = usersRecordedDates.sort((a,b)=> new Date(a.date) - new Date(b.date));
```
Comment (AndrewCerveny) [inline src/scripts.js:138]: Maybe change this const name form minToMax

**Review by AndrewCerveny — APPROVED** [review]
The changes look good. 
- Maybe rename the min constant method for the the calendar function. 
-delete that media query maybe save it on something else for a reference for later

File: src/scripts.js (hunk 1 above)
Comment (AndrewCerveny) [reply src/scripts.js:138]: renaming the function for restric Min Max

**Pull request merged**